guides/source/active_record_multiple_databases.md

```diff
@@ -150,9 +150,9 @@ config.active_record.writing_role = :default
 config.active_record.reading_role = :readonly
 ```
 
-It's important to connect to your database in a single model and then inherit from that model
+It's essential to connect to your database in a single model and then inherit from that model
```

Comment:
I think important is the correct word here. You *can* do it on individual models, it's just a bad idea.

Reply:
but the "essential" carries a stronger sense of necessity.

Reply:
I think "essential" implies you have no other option. That isn't true. But 🤷  it's not a deal breaker.

Reply:
Makes sense

Reply:
@ghiculescu Updated the changes.